hastory-cli/src/Hastory/Cli/OptParse/Types.hs

```diff
@@ -146,10 +144,9 @@ data SuggestAliasSettings =
   SuggestAliasSettings
   deriving (Show, Eq)
 
-data Settings =
+newtype Settings =
   Settings
     { setCacheDir :: Path Abs Dir
-    , remoteStorageClientInfo :: Maybe RemoteStorageClientInfo
```

Comment:
100% agree. Let me get back to this comment.

Reply:
@NorfairKing Syd - I think these should be addressed on their own merits. I have created #35 and #36. I'm happy to take them on in due time. (Likely very soon since these seem like obvious deficiencies in our application).